docker-compose.yaml

```diff
@@ -0,0 +1,58 @@
+version: "3.9"
+ 
+services:
+  app:
+    build:
+      context: .
+    environment:
+      SEQREPO_ROOT_DIR: /usr/local/share/seqrepo/2024-02-20
+      AWS_ACCESS_KEY_ID: 'DUMMYIDEXAMPLE'
+      AWS_SECRET_ACCESS_KEY: 'DUMMYEXAMPLEKEY'
+      AWS_DEFAULT_REGION: 'us-west-2'
+      GENE_NORM_DB_URL: http://dynamodb-local:8000
+ 
+    links:
+     - "dynamodb-local"
+    
+    ports:
+      - "8001:80"
+    
+    volumes:
+      - seqrepo_vol:/usr/local/share/seqrepo
+    command: >
+      sh -c "pipenv run gene_norm_update --update_all --update_merged &&
+             cd src wait_for_db &&
```

Comment:
wait_for_db is the custom function to handle the database race condition , as depends on only checks whether service is running or not. It will be implemented in next phase. Please review further and I will remove this command for now along with other review comments(if any) for this phase.

Reply:
@rajatkapoordfci Would you be able to update the PR description with what this phase intended to achieve and what the next phase(s) include?

Reply:
Done. Add Phase scope in the description.

Reply:
Thank you!